docs/team/peiyuan.md

```diff
@@ -2,33 +2,134 @@
 layout: page
 title: Lim Peiyuan's Project Portfolio Page
 ---
+### Introduction
 
-### Project: Financial Advisor Smart Tracker (FAST)
+I am a software developer for Financial Advisor Smart Tracker (FAST). I was in-charge of the Code Quality of FAST.
+At the time of developed FAST, I was a Year 2 Computer Science Undergraduate.
 
-Financial Advisor Smart Tracker (FAST) is a desktop address book application targeted at financial advisors, for
+---
+
+### Project Overview: Financial Advisor Smart Tracker (FAST)
+
+Financial Advisor Smart Tracker (FAST) is a desktop application targeted at financial advisors, for
 them to keep track of their clients as contacts. The user interacts with it using a CLI, and it has a GUI created with
-JavaFX. It is written in Java, and has about 10 kLoC.
+JavaFX. It is written in Java, and has about 12k LoC.
```

Comment:
Not sure if this is relevant, but should we stick to the old format `10 kLoc` or `10k Loc`?